src/components/views/DownloadModModal.vue

```diff
@@ -136,6 +136,11 @@ let assignId = 0;
 
         static allVersions: [number, DownloadProgress][] = [];
 
+
+        get activeDownload() {
```

Comment:
To build on my earlier similar comment about a progress-getter returning undefined or number: due to the poor type support of the Vuex version we're using, this will now return `any` type, which makes it difficult to detect errors that occur if the value is actually `undefined`. So

a) always check that these getters have a properly inferred return type. If Vuex is involved, changes are it doesn't. If the inferred type is wrong, provide correct return type manually (it's IMO ok to always do this to avoid bugs from future changed
b) Unless a getter like this can provide a sane fallback value, make sure the return type include `undefined` (or whatever the correct type is) so incorrect usage elsewhere causes an error.